s3/Package.md

```diff
@@ -1,333 +1,21 @@
-import ballerinax/aws.s3;
-
-configurable string accessKeyId = ?;
-configurable string secretAccessKey = ?;
-configurable string region = ?;
-configurable string bucketName = ?;
-
-s3:ClientConfiguration amazonS3Config = {
-    accessKeyId: accessKeyId,
-    secretAccessKey: secretAccessKey,
-    region: region
-};
-
-s3:Client amazonS3Client = check new (amazonS3Config);
-
-public function main() returns error? {
-    var getObjectResponse = amazonS3Client->getObject(bucketName, "test.txt");
-    if (getObjectResponse is s3:S3Object) {
-        log:printInfo(getObjectResponse.toString());
-        byte[]? byteArray = getObjectResponse["content"];
-        if (byteArray is byte[]) {
-            string content = check strings:fromBytes(byteArray);
-            log:printInfo("Object content: " + content);
-        }
-    } else {
-        log:printError("Error: " + getObjectResponse.toString());
-    }
-}
-```
-
-### Delete an object
-
-This sample shows how to delete an object that is available in a bucket. The bucket name and object name are required to do this operation. This operation returns an `error` if unsuccessful
-
-```ballerina
-import ballerina/log;
-import ballerinax/aws.s3;
-
-configurable string accessKeyId = ?;
-configurable string secretAccessKey = ?;
-configurable string region = ?;
-configurable string bucketName = ?;
-
-s3:ClientConfiguration amazonS3Config = {
-    accessKeyId: accessKeyId,
-    secretAccessKey: secretAccessKey,
-    region: region
-};
-
-s3:Client amazonS3Client = check new(amazonS3Config);
-
-public function main() {
-    error? deleteObjectResponse = amazonS3Client->deleteObject(bucketName, "test.txt");
-    if (deleteObjectResponse is error) {
-        log:printError("Error: " + deleteObjectResponse.toString());
-    } else {
-        log:printInfo("Successfully deleted object");
-    }
-}
-```
-
-### Delete a bucket
-
-This sample shows how to delete a bucket that is available in an user's account. The bucket name is required to do this operation. This operation returns an `error` if unsuccessful
-
-```ballerina
-import ballerina/log;
-import ballerinax/aws.s3;
-
-configurable string accessKeyId = ?;
-configurable string secretAccessKey = ?;
-configurable string region = ?;
-configurable string bucketName = ?;
-
-s3:ClientConfiguration amazonS3Config = {
-    accessKeyId: accessKeyId,
-    secretAccessKey: secretAccessKey,
-    region: region
-};
-
-s3:Client amazonS3Client = check new (amazonS3Config);
-
-public function main() {
-    error? deleteBucketResponse = amazonS3Client->deleteBucket(bucketName);
-    if (deleteBucketResponse is error) {
-        log:printError("Error: " + deleteBucketResponse.toString());
-    } else {
-        log:printInfo("Successfully deleted bucket");
-    }
-}
-```
\ No newline at end of file
+## Useful links
+- Discuss code changes of the Ballerina project in [ballerina-dev@googlegroups.com](mailto:ballerina-dev@googlegroups.com).
```

Comment:
Replace "in" with "via".

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-aws.s3/pull/55/commits/df2a25cd27ea5b50683e8473e083cb5f4a90927b